integration/__specs__/modules/components/cartPage/cartModal/cartPriceInput/cartPriceInput.ts

```diff
@@ -0,0 +1,11 @@
+import { Component } from '@Core/component';
+
+export class CartPriceInput extends Component {
+    protected selectors = {
+        priceInput: './/input[@name="price"]',
+    };
+
+    public async getNameInput() {
+        await this.element.waitForXpath(this.selectors.priceInput);
+    }
+}
```

Comment:
We don't need this component, since we create them using 'Component' in cartModal.ts